frame/transaction-payment/asset-conversion-tx-payment/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "pallet-asset-conversion-tx-payment"
+version = "4.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+description = "Pallet to manage transaction payments in assets by utilising the assets conversion pallet"
```

Comment:
Is it directly coupled with assets-conversion?
Should not, probably depends on a trait

Reply:
We use the `NativeOrAssetId` and `NativeOrAssetIdConverter` from the asset-conversion pallet for tests